config_test.go

```diff
@@ -672,6 +674,68 @@ func TestLoadConfig(t *testing.T) {
 	}
 }
 
+func TestGetEnvFunction(t *testing.T) {
+	// Rendering Template with placeholders
+	os.Setenv("PORT", "8082")
+	os.Setenv("TEST_ENV_VAR", "VALUES")
+	env_vars := envToMap()
+	fmt.Print(env_vars["PORT"])
+	if strings.Compare(env_vars["PORT"], "8080") == 0 && strings.Compare(env_vars["TEST_ENV_VAR"], "VALUES") == 0 {
+		t.Error("Get Env function not working as expected")
+	}
+
+}
+
+func TestRenderVariableTemplate(t *testing.T) {
+	// Rendering Template with placeholders
+	os.Setenv("PORT", "8082")
+	ConfigFile = "./integration/templatej8a1.yml"
+	config := new(Config).load()
+	if config.Connection.Downstream.Http.Port != 8082 {
```

Comment:
this should fail and not return valid config as opposed to simply having an empty port. otherwise it will
leave server in invalid state.